server/js/sockets.js

```diff
@@ -244,13 +245,21 @@ sockets.init = function socketInit(server) {
               PUBLIC_MQTT_CLIENT.publish(DAS.data, payloadString);
             }
             break;
-
+          case BOOST.achieved_max_speed:
+            socket.emit('power-model-running');
+            socket.emit(
+              'power-model-achieved-max-speed',
+              queryStringToJson(payloadString),
+            );
+            socket.emit('boost/achieved_max_speed', payloadString);
```

Comment:
Maybe consider using dashes here rather than slashes and underscores, only for consistency with other channels that we have

Reply:
I'm kind of with this as it's a good way to indicate that the data coming in is straight from the topic. Might consider changing the other topics to match, and then possibly also use `topics.js` in client.

Reply:
That's a good point.

Reply:
wait do you guys mean changing all topics in topics.js bc they all consist of slashes and underscores ??
so basically any topic used in dashboard folder must only have dashes ? @hallgchris @haotongwang

Reply:
The topics in `topics.js` correspond to MQTT topics, the ones we currently use in `client` are `socket.io` channel topics.
Quick reminders
- MQTT: Connects different parts of the system (DAS, Rapicam, Dashboard)
- socket.io: Dashboard server <-> client

Currently what we have is that MQTT uses slashes and underscores (hence the `topics.js`) and `socket.io` uses just dashes. However, we have been slowly transitioning to have `socket.io` channels correspond to the MQTT topics as the server just forwards the payload, resulting in many of the topics just differ in using slashes/underscores or dashes.

The suggestion here is to change the `socket.io` channels to also use slashes and underscores for channels that just receive payloads from the corresponing MQTT topic.
As for what you can do, probably leave this as is for now, and if you feel like it, make another PR changing the rest of the channels.

Hope that makes sense.

Reply:
thanks for the explanation, it kinda make same but doesnt at the same time.
I'll see what I can do about it and hopefully we can further discuss it in meeting :) @haotongwang